variables.tf

```diff
@@ -4,139 +4,269 @@ variable "name" {
 }
 
 variable "ses_configuration_arn" {
-  type = string
+  description = "SES configuration ARN"
+  type        = string
 }
 
 variable "ses_from_email" {
-  type = string
+  description = "From email address for SES"
+  type        = string
 }
 
 variable "auth_sms_message" {
-  type    = string
-  default = "Este código te lo envía {####}"
+  description = "Authentication SMS message"
+  type        = string
+  default     = "Este código te lo envía {####}"
 }
 
-variable "domain" {
-  type = string
+variable "html_email_message" {
+  description = "Custom HTML email message"
+  type        = string
+  default     = ""
 }
 
-variable "certificate_arn" {
-  type = string
+variable "email_verification_subject" {
+  description = "Email verification subject"
+  type        = string
 }
 
-variable "region" {
-  type = string
+variable "auto_verified_attributes" {
+  description = "List of attributes to be auto-verified in Cognito"
+  type        = list(string)
+  default     = []
+}
+
+
+variable "enable_software_token_mfa" {
+  description = "Boolean to enable/disable software token MFA configuration"
+  type        = bool
+  default     = false
+}
+
+variable "email_sending_account" {
+  description = "Email sending account"
+  type        = string
+  default     = "DEVELOPER"  # Valor por defecto razonable
+}
+
+variable "reply_to_email_address" {
+  description = "Reply-to email address"
+  type        = string
+  default     = "admin@default.com"  # Valor por defecto razonable
+}
```

Comment:
Esta variable debe ser obligatoria porque si el email no coincide con la identidad de ses, el deploy del modulo fallará

Reply:
The reply_to_email_address variable has been corrected to make it mandatory, removing its default value. This ensures that the provided email matches the SES identity and prevents module deployment failures.